src/lib/stepper/stepper.spec.ts

```diff
@@ -457,6 +485,57 @@ function checkStepHeaderBlur(stepHeaderEl: HTMLElement, fixture: ComponentFixtur
   expect(stepHeaderEl.blur).toHaveBeenCalled();
 }
 
+function checkEditableStep(stepperComponent: MdStepper,
```

Comment:
Need function descriptions. I'd also prefer to have the functions names start with `assert` instead of `check`, and potentially making the function names more specific about what they do